Content.Client/DoAfter/DoAfterOverlay.cs

```diff
@@ -152,9 +150,15 @@ protected override void Draw(in OverlayDrawArgs args)
                 }
 
                 var xProgress = (EndX - StartX) * elapsedRatio + StartX;
-                var box = new Box2(new Vector2(StartX, 3f) / EyeManager.PixelsPerMeter, new Vector2(xProgress, 4f) / EyeManager.PixelsPerMeter);
+                var box = new Box2(new Vector2(StartX, 2f) / EyeManager.PixelsPerMeter, new Vector2(xProgress, 5f) / EyeManager.PixelsPerMeter); // Stellar - Revamped DoAfters
+                var boxGlow = new Box2(new Vector2(StartX - 1f, 1f) / EyeManager.PixelsPerMeter, new Vector2(xProgress + 1f, 6f) / EyeManager.PixelsPerMeter); // Stellar - Revamped DoAfters
+                var boxHighlight = new Box2(new Vector2(StartX, 3f) / EyeManager.PixelsPerMeter, new Vector2(xProgress, 4f) / EyeManager.PixelsPerMeter); // Stellar - Revamped DoAfters
```

Comment:
try not to have hardcoded numbers, these should be at the minimum named constants that indicate what they mean, if it makes sense, they could be datafields on the ui theme as well